### PR TITLE
Updated Bloom Recipe and Added TFC Ore Piece Extraction Recipes

### PR DESCRIPTION
**Pull request by Nyxane**
Made it so the bloom recipes require a minimum temp to work/progress

### CODE REVIEW

**Comment by pietro-lopes** [comment]
is this needed?
`Item.of('tfc:refined_iron_bloom').getId()`
you already have id with the name itself

**Comment by Nyxane** [comment]
No, it was just an artifact from copying my last function, figured it wouldn't matter :p

File: kubejs/server_scripts/recipes/add.js [hunk 1]
```diff
@@ -151,16 +151,34 @@ let recipeAdd = (/** @type {Internal.RecipesEventJS} */ event) => {
...
+  event.recipes.create.pressing(
+    [
+      { item: "tfc:metal/ingot/wrought_iron" }
+    ],
+    [
+      {
+        type: 'tfc:heatable',
+        min_temp: 921,
+        ingredient: {
+          item: Item.of('tfc:refined_iron_bloom').getId(),
```
Comment (pietro-lopes) [inline kubejs/server_scripts/recipes/add.js:177]:
```suggestion
          item: 'tfc:refined_iron_bloom',
```

File: kubejs/server_scripts/recipes/add.js [hunk 2]
```diff
@@ -151,16 +151,34 @@ let recipeAdd = (/** @type {Internal.RecipesEventJS} */ event) => {
...
+  event.recipes.create.pressing(
+    [
+      { item: "tfc:refined_iron_bloom" }
+    ],
+    [
+      {
+        type: 'tfc:heatable',
+        min_temp: 921,
+        ingredient: {
+          item: Item.of('tfc:raw_iron_bloom').getId(),
```
Comment (pietro-lopes) [inline kubejs/server_scripts/recipes/add.js:163]:
```suggestion
          item: 'tfc:raw_iron_bloom',
```

**Review by pietro-lopes — CHANGES_REQUESTED** [review]

**Review by pietro-lopes — APPROVED** [review]

**Pull request merged**